crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -2846,10 +2846,13 @@ impl AuthorityPerEpochStore {
         // We track transaction execution cost separately for regular transactions and transactions using randomness, since
         // they will be in different PendingCheckpoints.
         let tables = self.tables()?;
-        let per_commit_budget = self
+        let default_per_commit_budget = self
             .protocol_config()
             .max_accumulated_txn_cost_per_object_in_mysticeti_commit_as_option()
             .unwrap_or(0);
+        let gas_budget_based_txn_cost_absolute_cap_commit_multiple = self
```

Comment:
`multiplier` or `factor` might be a better word here, as `multiple` makes me think it contains multiple caps

Reply:
I wanted to express that it indicates a number of commits' worth of budget, multiplier or factor seem more generic and don't express that as well IMO. How about `commit_count`, do you think that's any better or still confusing?